examples/custom/README.md

```diff
@@ -1,3 +1,3 @@
 # Account Custom Settings example
 
-An example of custom settings being applied to the account. This use case may be helpful in development environments.
+An example of custom settings being applied to the account. This use case may be helpful in development environments. This example consumes the experimental options submodule that uses the RestAPI provider to call an experimental API that is currently unsupported.
```

Comment:
Can we be more specific as to the options the experimental submodule is going to set in the example?